recipes/default.rb

```diff
@@ -17,17 +17,29 @@
 # limitations under the License.
 #
 
-# apt_repository 'lacework' do
-#   source "#{download_url}/latest/DEB/Ubuntu/16.04 xenial main"
-#   source "#{download_url}/latest/DEB/Debian/9 whatever main"
+dist = node['platform']
+pver =
+  if node['platform'] == 'debian'
+    node['platform_version'].to_i
```

Comment:
Is the point of this to get just the major version?

Reply:
Yes.